tests/device_tests/test_debuglink.py

```diff
@@ -63,7 +63,9 @@ def load_device():
         )
 
     # start from a clean slate:
-    client.debug.reseed(0)
+    resp = client.debug.reseed(0)
+    if isinstance(resp, messages.Failure):
```

Comment:
let's be a bit stricter, and use xfail instead of skip:
```
if isinstance(resp, messages.Failure) and not isinstance(client.transport, UdpTransport):
	pytest.xfail("reseed only supported on emulator")
```

Reply:
Good idea, fixed in 26f0dc6a99ee6e41e388476ec8a5198774e29620.